src/util/KVStore/AMOStore.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Jack Khuu, Justin Johnson, Lukas Joswiak
+
+#include "AMOStore.hpp"
+
+#include <stdlib.h>
+
+#include <map>
+#include <string>
+#include <iostream>
+
+#include "Action.hpp"
+#include "AMOCommand.hpp"
+#include "AMOResponse.hpp"
+#include "KVStore.hpp"
+
+namespace KVStore {
+using std::string;
```

Comment:
Remove using statements.

Reply:
Hard disagree

Using is bad style for header files. It is fine in src files